### PR TITLE
label for max rows

### PR DESCRIPTION
**Pull request by benrudolph**
for icds

cc: @orangejenny

### CODE REVIEW

File: corehq/apps/export/templates/export/partial/export_list_controller.html [hunk 1]
```diff
@@ -2,9 +2,17 @@
...
 {% load hq_shared_tags %}
 {% load djangular_tags %}
+{% load humanize %}
 
 <div ng-controller="ListExportsController"
      ng-cloak>
+    {% if request|toggle_enabled:"PAGINATED_EXPORTS" %}
+    <div class="alert alert-info">
+        {% blocktrans with max_rows=max_exportable_rows|intcomma %}
+        These exports are paginated. That means exports that have over {{ max_rows }} rows will be split into multiple files.
```
Comment (orangejenny) [inline corehq/apps/export/templates/export/partial/export_list_controller.html]: Should this be "over {{ max rows }} rows, *they* will be split"?

**Comment by orangejenny** [comment]
Curious why this is behind a flag - are there domains that have exports this big but do need them to be in a single file?

**Comment by benrudolph** [comment]
normally you can't export over 100k rows, and we don't want to make that a thing. however we have an exception for icds, and since it is greater than 100k rows we need to split it into multiple files

File: corehq/apps/export/templates/export/partial/export_list_controller.html (hunk 1 above)
Comment (benrudolph) [inline corehq/apps/export/templates/export/partial/export_list_controller.html]: i think i'd have to change the beginning of the sentence for that, perhaps it doesn't read that clearly 😞

Comment (orangejenny) [inline corehq/apps/export/templates/export/partial/export_list_controller.html]: oh, I misread this, looking again it sounds fine

**Review by orangejenny — APPROVED** [review]

**Pull request merged**